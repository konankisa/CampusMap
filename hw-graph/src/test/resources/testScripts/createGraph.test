# Creates a graph. Initially has no nodes and edges

# Create a graph
CreateGraph graph