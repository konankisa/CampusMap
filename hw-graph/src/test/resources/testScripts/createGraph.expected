# Creates a graph. Initially has no nodes and edges

# Create a graph
created graph graph